File: python-problem-solving/01.found_nemo_func.py
import time

nemo = ['cloud','door','fish','nemo']

def findNemo(array):
    start_time = time.time()
    for i in array:
        if i == 'nemo':
            print('Found Nemo')
    print((time.time() - start_time))
findNemo(nemo)

File: js-problem-solving/01_found_nemo_func.js
const nemo = ['cloud','door','fish','nemo'];

const large = new Array(100).fill('nemo');

function findNemo(array) {
    for(i=0; i<array.length; i++){
        if (array[i] == 'nemo'){
            console.log('Found Nemo!');
        }
    }
}

findNemo(large);











